less-print/c2corg_ui.less

```diff
@@ -1,13 +0,0 @@
-@import "viewdetails.less";
```

Comment:
I was playing around..i didn't even notice it had an impact :P I will bring it back.